src/client/sandbox/cookie/index.js

```diff
@@ -139,9 +106,8 @@ export default class CookieSandbox extends SandboxBase {
     }
 
     setCookie (document, cookie, syncWithServer) {
-        // eslint-disable-next-line no-restricted-properties
-        if (cookie.length > BYTES_PER_COOKIE_LIMIT || destLocation.getParsed().protocol === 'file:')
-            return cookie;
+        if (this._isCannotSetCookie())
```

Comment:
Add comment for this line about covered behavior.

We usually don't use function names with negation.
So rewrite as
```js
if (!this._isCanSetCookie())
    return;
```